docs/walkthrough/walkthrough-istio.md

```diff
@@ -0,0 +1,309 @@
+# Tekton Dashboard walkthrough - Kind and Istio
+
+This guide walks you through installing a working Tekton Dashboard locally from scratch. It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [Service mesh benefits](#service-mesh-benefits)
+* [Creating a local Kubernetes cluster](#creating-a-local-kubernetes-cluster)
+* [Installing the istio operator](#installing-the-istio-operator)
+* [Installing the istio service mesh](#installing-the-istio-service-mesh)
+* [Note on sidecar injection](#note-on-sidecar-injection)
+* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Installing Tekton Triggers (optional)](#installing-tekton-triggers-optional)
+* [Installing Tekton Dashboard](#installing-tekton-dashboard)
+* [Setting up the istio Gateway and VirtualService](#setting-up-the-istio-gateway-and-virtualservice)
+* [Browsing the istio dashboard](#browse-the-istio-dashboard)
+* [Cleaning up](#cleaning-up)
+
+## Before you begin
+
+Before you begin, make sure the following tools are installed:
+
+1. [`kind`](https://kind.sigs.k8s.io): For creating a local cluster running on top of docker.
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl): For interacting with your kubernetes cluster.
+1. [`istioctl`](https://istio.io/latest/docs/setup/getting-started/#download): For installing istio.
+
+Nothing more is required to run the walkthrough !
+
+## Service mesh benefits
+
+Using a service mesh brings benefits in several domains:
+- security
+- observability
+- resiliency
+
+Some security benefits:
+- mutual TLS brings secure communications inside the cluster (encrypted and authenticated)
+- egress control prevents data exfiltration and domains white list
+- network policies allow to secure pod-to-pod or service-to-service communication at the network and application layers
+
+Some observality benefits:
+- understand how service performance impacts things upstream and downstream
+- help detect and fix issues quickly and efficiently
+
+Some resiliency benefits:
+- automatically retry failed service calls
+- mirror traffic to send a copy of live traffic to a mirrored service
+- add circuit breakers directly on the wire
+- inject network faults to test applications behaviour
+
+The list is by no means exhaustive and service meshes bring a lot more benefits than just those enumerated above.
+
+Visit [istio's webpage](https://istio.io/latest/docs/concepts/what-is-istio/) or [istio's blog](https://istio.io/latest/blog) for more infos.
```

Comment:
small typo, otherwise this is a great summary! 🎉 

```suggestion
Visit [istio's webpage](https://istio.io/latest/docs/concepts/what-is-istio/) or [istio's blog](https://istio.io/latest/blog) for more info.
```

Reply:
Fixed